### ISSUE TITLE
Label field problem

### ISSUE THREAD

**Issue by PabloManuel78** [kevinchappell/formBuilder#1286]
### Environment Details: PHP8.1+JQUERY+APACHE in PLESK

 - formBuilder Version: Version: 3.7.4
 - Browser: Chrome
 - OS: Windows 10

### Expected Behavior
Be able to edit the label text of new and existing forms

### Actual Behavior
I can't edit the field labels.

### Screenshot - *(optional)*
![image](https://user-images.githubusercontent.com/5701864/159916245-7ff401bd-dd25-48e2-9bc9-f77a01a495c2.png)
![image](https://user-images.githubusercontent.com/5701864/159916290-b25aa14e-4181-4209-8315-e37598609c2c.png)

It's been going on for a few days. Suddenly I can't edit the label of the form fields.

Internally the tag is not an "input", it is a <DIV> and it is not editable.

Bookstore charge from:
<script src="https://formbuilder.online/assets/js/form-builder.min.js"></script>

<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>

<link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/themes/smoothness/jquery-ui.css">
<script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js"></script>

**Comment by Jojoshua** [comment]
This should have been fixed in the latest version
